traffic_portal/app/src/modules/private/cdns/config/config.tpl.html

```diff
@@ -26,98 +26,272 @@
         </ol>
         <div class="pull-right" ng-show="!settings.isNew">
             <button type="button" class="btn btn-default" ng-click="navigateToPath('/cdns/' + cdn.id)">Cancel</button>
-            <button class="btn btn-primary" title="Snapshot {{cdn.name}} Config" ng-click="confirmSnapshot(cdn)"><i class="fa fa-camera"></i>&nbsp;&nbsp;Perform Snapshot</button>
+            <button class="btn btn-primary" title="Perform {{cdn.name}} Snapshot" ng-click="confirmSnapshot(cdn)"><i class="fa fa-camera"></i>&nbsp;&nbsp;Perform Snapshot</button>
```

Comment:
Default `button[type=submit]` is inappropriate here; should be `button[type=button]`. Good job changing that title too btw - I can't believe I never noticed that before.